wrappers/services.go

```diff
@@ -348,8 +348,9 @@ func StartServices(apps []*snap.AppInfo, disabledSvcs []string, flags *StartServ
 			continue
```

Comment:
this is not related to this PR but I was looking at StartServices to review it and I'm wondering shouldn't disabledSvcs also affects the activators?

Reply:
ah, this is fixed in the previous PR?

Reply:
yes precisely, it's fixed in the other services PR- because fixing that issue ending up revealing an issue during refreshes which is fixed in the same PR.